generator/script/CustomRenderer.js

```diff
@@ -14,7 +14,7 @@ exports.render_upgrade = function (color) {
         if (href === null) {
             return text;
         }
-        let out = `<a class="links" style="${color}" href="${href}"`;
+        let out = `<a class="links" style="color :${color[1]}" href="${href}"`;
```

Comment:
you need to map the `1` to a constant like `MAIN_COLOR_INDEX= 1` to improve readability and maintainability